CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## next
 
+* Added `display_name` DSL to `ResourceDefinition` and `MediaType`
+  * It is a purely informational field, mostly to be used by consumers of the generated docs
+  * It defaults to the class name (stripping any of the prefix modules)
+* Revamped document generation to output a more compact format:
+    * 1 file per api version: including info, resources, schemas and traits.
+    * 1 single index file with global info plus just a list of version names
+    * new task currently usable through `bundle exec rake praxis:docs:experiments`
+      * NOTE: leaves the current doc generation tasks and code intact (until the doc browser is switched to use this)
+* Specialized `Multipart`’s family in its description to be ‘multipart’ instead of ‘hash’.
+* Added `Praxis::Handlers::FormData` for 'multipart/form-data'. Currently returns the input unchanged in `parse` and `generate`. 
+* Added `Praxis::Handlers::WWWForm` for form-encoded data.
+* Added `Docs::Generator`, experimental new documentation generator. Use the `praxis:docs:generate_beta` rake task to generate. *Note*: not currently compatible with the documentation browser.
+* Added `Application.validation_handler` to customize response generation for validation errors. See [validation_handler.rb](lib/praxis/validation_handler.rb) for default version.
```

Comment:
This line is incorrect, it might be a result of a merge/rebase...etc. It already exists in 0.17.0